src/lib/components/workflow/dropdown-filter/workflow-datetime-filter.svelte

```diff
@@ -80,7 +98,12 @@
         operator: '',
         parenthesis: '',
       };
-      $workflowFilters = [...getOtherFilters(), filter];
+      $workflowFilters = [...getOtherFilters($workflowFilters), filter];
+      $persistedWorkflowFilters = [
```

Comment:
**Note:** This seems like the most simple solution for the time being in order to only persist the filters we actually want (and not ones that are applied from a query on a link), but open to ideas about better ways to update `persistedWorkflowFilters` based on `workflowFilters` without repeating logic 💡

Reply:
I think this is totally fine for now, but when we implement our actions/reducers, we'll want to consolidate this logic under a single action. This will be hard to maintain and grok all the places we need to set these filters and how/when to keep them in sync.